### PR TITLE
Add overview based on Michael's comment

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

File: content/concepts/architecture.md [hunk 1]
```diff
@@ -13,6 +13,24 @@ This page tells you the architecture and basic concepts in open-cluster-manageme
 
 <!-- spellchecker-enable -->
 
+## Overview
+
+open-cluster-management enables any capability within the Kubernetes ecosystem to orchestrate itself across multiple clusters and cloud providers. Consider the requirements that a capability within the Kubernetes ecosystem would need to become multicluster-aware. 
+
+- The service must have an API to reason about the inventory of available clusters.
```
Comment (juliana-hsu) [inline content/concepts/architecture.md:20]: The service must have an API to determine the inventory of available clusters.

File: content/concepts/architecture.md [hunk 2]
```diff
@@ -13,6 +13,24 @@ This page tells you the architecture and basic concepts in open-cluster-manageme
 
 <!-- spellchecker-enable -->
 
+## Overview
+
+open-cluster-management enables any capability within the Kubernetes ecosystem to orchestrate itself across multiple clusters and cloud providers. Consider the requirements that a capability within the Kubernetes ecosystem would need to become multicluster-aware. 
+
+- The service must have an API to reason about the inventory of available clusters. 
+- The service must have a way to reason about where to schedule and assign Kubernetes API manifests to a selected set of clusters.
```
Comment (juliana-hsu) [inline content/concepts/architecture.md:21]: The service must have a way to determine where to schedule and assign Kubernetes API manifests to a selected set of clusters.

File: content/concepts/architecture.md [hunk 3]
```diff
@@ -13,6 +13,24 @@ This page tells you the architecture and basic concepts in open-cluster-manageme
...
+- The service must have a way to deliver desired Kubernetes API manifests to a selected set of clusters. 
+- The service must have a way to govern how users access available clusters or groups of clusters in the fleet. 
+- Optionally, the service may need to extend the management agent with additional built-in controllers that should be run on managed clusters.
+
+open-cluster-management provides core primitives to satisfy the above requirements to ease the multicluster-aware enablement:
```
Comment (juliana-hsu) [inline content/concepts/architecture.md:26]: open-cluster-management provides core primitives to satisfy the above requirements to ease the multicluster enablement:

**Comment by chenz4027** [comment]
@qiujian16 I have a PR open to fix the issues that Juliana proposed: https://github.com/qiujian16/ocm-website/pull/1
Could you help merge it to this branch? Thanks!

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**